google/cloud/forseti/common/util/logger.py

```diff
@@ -52,7 +52,6 @@ def get_logger(module_name):
     syslog_handler.setFormatter(logging.Formatter(SYSLOG_LOG_FMT))
 
     logger_instance = logging.getLogger(module_name)
-    logger_instance.addHandler(console_handler)
```

Comment:
I'm not sure we want to remove the console handler, rather it's noisy at it's current level, e.g. it should only be taking INFO logging statements.

Reply:
By default it was showing only INFO logs, we have log statements when reading variables / processing data. I don't think this is what the user wants to see and we are planning to catch all the exceptions and not display any stack trace in the console however the stack trace will be logged to stackdriver - if we enable printing logs to console, It's hard for us to both log the error to stackdriver and not show it to the user.